README.md

```diff
@@ -4,34 +4,40 @@
 [![Client Build Status](../../workflows/Client%20Angular/badge.svg?branch=master)](../../actions?query=workflow%3A"Client+Angular")
 [![End to End Build Status](../../workflows/End-to-End/badge.svg?branch=master)](../../actions?query=workflow%3AEnd-to-End)
 
-- [Setup](#setup)
-  - [Open the project in VS Code](#open-the-project-in-vs-code)
-  - [Installing the client dependencies](#installing-the-client-dependencies)
-  - [Enable ESLint in VS Code](#enable-eslint-in-vs-code)
-- [Running your project](#running-your-project)
-- [Testing and Continuous Integration](#testing-and-continuous-integration)
-  - [Testing the client](#testing-the-client)
-    - [Linting the client](#linting-the-client)
-  - [Testing the server](#testing-the-server)
-  - [End to end testing with Cypress](#end-to-end-testing-with-cypress)
-  - [GitHub Actions](#github-actions)
-- [Resources](#resources)
-  - [Angular](#angular)
-  - [Cypress (end-to-end testing)](#cypress-end-to-end-testing)
-
-During this lab, you will use a todo API like you created in the previous lab
-by building a basic client-side application using Angular. This will enable you
-to better handle user input and display data returned from the server. As always, 
-you'll be expected to make good use of the version control (e.g., branching for features and merging changes to the master branch as appropriate) and project management 
-tools available to you: write good commit messages, test things, document issues, etc.
+* [Setup](#setup)
```

Comment:
I know you prefer `*` for unordered lists @NicMcPhee but there are some good arguments for using `-`, mainly that it doesn't require a modifier key to type. `-` is what Prettier prefers and when the document is formatted with it will convert all lists to `-`.

Reply:
Yeah, I realized part way through that Prettier and I were going to have fights, so I stopped converting everything over. I'll need to modify the configuration of my ToC tool, because it is converting those to '*' instead of '-'. I found that setting and switched to '-' and will start moving everything back to that. (Although Prettier may do a lot of that for us.)